### PR TITLE
allow verbose logging

### PR DESCRIPTION
**Pull request by johanneskastl**

### CODE REVIEW

**Comment by EugenMayer** [comment]
Besides you need to upstream merge now, should we just expose 'extraArguments' and let the user add whatever needed via the helm chart? 
In anyway, i'am fine with -v being a option/flag too

**Comment by johanneskastl** [comment]
Branch was rebased and conflicts were solved.

I woud propose to merge this now, and if lots of other extra arguments appear we can always introduce a new variable, right?

File: charts/iperf3/values.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@ service:
   # externalIp:
   # the port to bind on the service, if enabled.
   port: 40000
+# uncomment for verbose logging
+# verbose: true
```
Comment (EugenMayer) [inline charts/iperf3/values.yaml:8]: please just uncomment and use the defaulft, so just use `verbose: false` - no need to uncomment it, makes it harder to read what is available and what is just a comment

**Review by EugenMayer — CHANGES_REQUESTED** [review]

File: charts/iperf3/values.yaml (hunk 1 above)
Comment (johanneskastl) [reply charts/iperf3/values.yaml]: Done, see latest rebase.

File: charts/iperf3/templates/deployment.yml [hunk 2]
```diff
@@ -26,6 +26,9 @@ spec:
             - -p 40000
             - -i 1
             - --forceflush
+{{- if .Values.verbose }}
+            - -V
+{{- end }}
```
Comment (EugenMayer) [inline charts/iperf3/templates/deployment.yml:31]:
```suggestion
            {{- if .Values.verbose }}
            - -V
            {{- end }}
```

Comment (EugenMayer) [reply charts/iperf3/templates/deployment.yml:31]: small suggesting for formatting

**Review by EugenMayer — CHANGES_REQUESTED** [review]
Thank you!

File: charts/iperf3/templates/deployment.yml (hunk 2 above)
Comment (johanneskastl) [reply charts/iperf3/templates/deployment.yml]: Thanks, fixed.

**Comment by EugenMayer** [comment]
Thank you for pushing it all through. Very appreciated

**Pull request merged**